blockware/fireworks/src/main.cpp

```diff
@@ -11,73 +11,163 @@
 #include <Particle.h>
 #include <DefaultConfig.h>
 
-#ifndef max
-#define max(X,Y) (((X) > (Y)) ? (X) : (Y))
-#endif
+#include <LIS2DW12Sensor.h>
+
 
 const int tlWidth = 40;
 const int tlHeight = 33;
-const float FIREWORK_CHANCE = 0.05;
-const int FIREWORKS_PER_FRAME = 64;
-const char *username = "@magusnn";
+const char *username = "@twitter";
 const char *years = "5 years";
 
 
 Adafruit_SSD1351 tft =
   Adafruit_SSD1351(SCREEN_WIDTH, SCREEN_HEIGHT, &SPI, CS_PIN, DC_PIN, RST_PIN);
 
-std::vector<unsigned char*> Logos;
```

Comment:
did getting rid of the array of logos free up the memory for the 200 particles we reserve? i know we hit a limit at 22 before and I guess the logos were where we got the extra space?

Reply:
The array didn't but not referencing all the logos did. Since this change only needs one bitmap the array wasn't needed.